src/AddIns/Uno.UI.Svg/SvgProvider.cs

```diff
@@ -0,0 +1,73 @@
+﻿#nullable enable
```

Comment:
I think this is not necessary, you already added `<Nullable>enable</Nullable>` in csproj